app/models/uphold_connection.rb

```diff
@@ -406,6 +408,17 @@ def find_or_create_uphold_card!
     end
   end
 
+  def has_deposit_capability?
+    result = connection_client.users.get_capability("deposits")
+
+    case result
+    when UpholdUserCapability
+      true
```

Comment:
Hmm, I'm not actually sure this is valid. There are additional checks based on requirements/restrictions that inform whether or not they have the capability.

Reply:
Following up with Uphold on this one